apps/nextjs-website/src/components/molecules/PartRenderer/PartRenderer.tsx

```diff
@@ -17,8 +17,6 @@ const PartRenderer = ({ part }: PartRendererProps): ReactNode | null => {
   switch (part.component) {
     case 'alert':
       return <AlertPart {...part} />;
-    case 'quote':
-      return <Quote {...part} />;
```

Comment:
This way, you will remove the quote part on each page. We just want to remove it from the Tutorial pages